src/registry/registry-json.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+// https://github.com/isaacs/tshy?tab=readme-ov-file#commonjs-dialect-polyfills
+// @ts-ignore suppress "Import assertions are not allowed on statements that compile to CommonJS 'require' calls."
+import registry from '@substrate/asset-transfer-api-registry/docs/registry.json' assert { type: 'json' };
```

Comment:
needs another release on https://github.com/paritytech/asset-transfer-api-registry to see if the change to esm style exports will fix this